src/submodules/pathfinding/astar_continuous.jl

```diff
@@ -0,0 +1,181 @@
+"""
+    find_continuous_path(pathfinder, from, to, model)
+Functions like find_path, but uses the output of find_path and converts it to the coordinate
```

Comment:
instead of find_path, type `find_path` (i.e. anything that is _code_ must be in backticks)